### PR TITLE
[CI] Fix test environment.

### PR DESCRIPTION
**Pull request by trivialfis**

### CODE REVIEW

**Comment by trivialfis** [comment]
The ubjson is removed from macos. I opened a PR there https://github.com/Iotic-Labs/py-ubjson/pull/20 see if there's response.

**Comment by trivialfis** [comment]
There's another failure in buidkite with cupy. Will look into it in a different PR.

File: tests/ci_build/Dockerfile.i386 [hunk 1]
```diff
@@ -1,7 +1,6 @@
 FROM i386/debian:sid
 
 ENV DEBIAN_FRONTEND noninteractive
-SHELL ["/bin/bash", "-c"]   # Use Bash as shell
```
Comment (hcho3) [inline tests/ci_build/Dockerfile.i386:4]: Can we bring the line back without the comment?
```
SHELL ["/bin/bash", "-c"]
```
This should work.

Comment (trivialfis) [reply tests/ci_build/Dockerfile.i386:4]: Done. Thank you for the suggestion. That's a weird bug.

**Review by hcho3 — APPROVED** [review]

**Comment by hcho3** [comment]
Merging to unblock the CI.

**Comment by trivialfis** [comment]
Thanks, was about to merge it. Wondering what's the issue with the freebsd test. Hasn't been the most stable action workflow.

**Comment by hcho3** [comment]
> Wondering what's the issue with the freebsd test. Hasn't been the most stable action workflow.

We may consider removing gtest from the FreeBSD tests.

**Comment by trivialfis** [comment]
Will do some debugging with column split. From limited experience, qemu tends to hang if an application oversubscribes the system threads, and column split related tests certainly oversubscribe the threads since all workers are using all available threads.

**Pull request merged**